src/analytics/executionEnvs.js

```diff
@@ -0,0 +1,71 @@
+const _ = require("lodash");
+
+const { getEventContextValues } = require("./eventContext");
+
+// Defines all non-local execution environemnts from which Wasp CLI sends
+// telemetry data to PostHog
+// - "contextKey" is used to identify the execution env in the event context
+// - "name" is used to display the env in the output
+const executionEnvs = {
+    replit: { contextKey: "replit", name: "Replit" },
+    gitpod: { contextKey: "gitpod", name: "Gitpod" },
+    codespaces: {
+        contextKey: "codespaces",
+        name: "GH Codespaces"
+    },
+    ci: { contextKey: "ci", name: "CI" }
+};
+
+// Organize events by the execution env: Replit, Gitpod, Github Codepsaces, CI, or the local usage.
+// We are most interested in local usage (user running Wasp on their computer), which is why we want
+// to do this separation, but we also do some analysis on the rest of events.
```

Comment:
```suggestion
// Organize events by the execution env:
//   - non-local: e.g. Replit, Gitpod, Github Codepsaces, CI, ... .
//   - local: User running Wasp on their computer.
```

Warning: nitpicking ahead!

Main thing I wanted to add here is make the list less of a "it is exactly these non local envs" and more "could be some of these", in order to make comment more robust to adding or removing execution envs in the future.

I also removed the info about us caring more about local -> I don't think it is really important here. It makes sense to separate it even without us caring about one more or less, they are fundamentally different.